golem/actions.py

```diff
@@ -130,6 +130,10 @@ def console_exit():
         pass
 
 
+def pdb_debug():
```

Comment:
What's the goal of this action? it probably is not enough descriptive for non python developers

Reply:
This shouldn't have been in there. I was playing with a change in my fork. Will remove. 
I've found you can just drop a pdb breakpoint in a test if you need it, so this change isn't needed.